src/plugins/data_table/server_side_data_table.js

```diff
@@ -70,13 +70,49 @@ var ServerSideDataTable = (function() {
 
     // dataTables callback to allow addition of params to the ajax request
     _addServerParams : function(aoData) {
+      if (this.simpleParams) {
+        var sortBy, sortDir, limit, start, requestId;
+
+        var indexOfSortedColumn = this._getDataTableParam(aoData, "iSortCol_0");
```

Comment:
as discussed we are leaving this for now.. if you need multi-sort, don't set simpleParams (and if & when that is coming up and we want simpleParams + multisort we can take on then)